docs/installation.md

```diff
@@ -2,422 +2,437 @@
+chosen operating system below.
 
 
-### *Installing dependencies on CentOS*
+### 3.1 Installing dependencies on CentOS
 
-Unless you've already done so, install Zonemaster *Engine* according to its own [installtion instruction](https://github.com/dotse/zonemaster-engine/blob/master/docs/installation.md).
-
-```sh
-sudo yum install perl-Module-Install perl-IO-CaptureOutput perl-String-ShellQuote
-sudo cpanm -i Config::IniFiles Daemon::Control JSON::RPC::Dispatch Parallel::ForkManager Plack::Builder Plack::Middleware::Debug Router::Simple::Declare Starman
+```sh 
+sudo yum install perl-Module-Install perl-IO-CaptureOutput perl-String-ShellQuote sudo cpanm -i Config::IniFiles Daemon::Control JSON::RPC::Dispatch Parallel::ForkManager Plack::Builder Plack::Middleware::Debug Router::Simple::Declare Starman 
 ```
 
-Install the chosen database engine and related dependencies, and start the database engine:
+Install the chosen database engine and related dependencies.
 
-* MySQL
+#### 3.1.1 MySQL
 
-  ```sh
-  sudo yum install wget
-  wget http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm
-  sudo rpm -ivh mysql-community-release-el7-5.noarch.rpm
-  sudo yum install mysql-server perl-DBD-mysql
-  sudo systemctl start mysqld
-  ```
-
-* PostgreSQL
+```sh 
+sudo yum install wget wget http://repo.mysql.com/mysql-community-release-el7-5.noarch.rpm 
+sudo rpm -ivh mysql-community-release-el7-5.noarch.rpm 
+sudo yum install mysql-server perl-DBD-mysql 
+sudo systemctl start mysqld 
+```
 
-  >
-  > At this time there is no instruction for using PostgreSQL on CentOS.
-  >
+#### 3.1.2 PostgreSQL
 
-* SQLite
+>
+> At this time there is no instruction for using PostgreSQL on CentOS.
+>
 
-  >
-  > At this time there is no instruction for using SQLite on CentOS.
-  >
+#### 3.1.3 SQLite
 
+>
+> At this time there is no instruction for using SQLite on CentOS.
+>
 
-### *Installing dependencies on Debian and Ubuntu*
 
-Unless you've already done so, install Zonemaster *Engine* according to its own [installtion instruction](https://github.com/dotse/zonemaster-engine/blob/master/docs/installation.md).
+### 3.2 Installing dependencies on Debian and Ubuntu
 
-```sh
-sudo apt-get update
-sudo apt-get install git libmodule-install-perl libconfig-inifiles-perl libdbd-sqlite3-perl starman libio-captureoutput-perl libproc-processtable-perl libstring-shellquote-perl librouter-simple-perl libclass-method-modifiers-perl libtext-microtemplate-perl libdaemon-control-perl 
-sudo cpanm -i Plack::Middleware::Debug Parallel::ForkManager JSON::RPC
+```sh 
+sudo apt-get update 
+sudo apt-get install git libmodule-install-perl libconfig-inifiles-perl libdbd-sqlite3-perl starman libio-captureoutput-perl libproc-processtable-perl libstring-shellquote-perl librouter-simple-perl libclass-method-modifiers-perl libtext-microtemplate-perl libdaemon-control-perl sudo cpanm -i Plack::Middleware::Debug Parallel::ForkManager JSON::RPC 
 ```

```

Comment:
It does not look correct. You have merged the two last lines. It should be

```sh
sudo apt-get update
sudo apt-get install git libmodule-install-perl libconfig-inifiles-perl libdbd-sqlite3-perl starman libio-captureoutput-perl libproc-processtable-perl libstring-shellquote-perl librouter-simple-perl libclass-method-modifiers-perl libtext-microtemplate-perl libdaemon-control-perl 
sudo cpanm -i Plack::Middleware::Debug Parallel::ForkManager JSON::RPC
```
as far as I can see.